presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -258,8 +258,8 @@ public int getMaxQueryRunningTaskCount()
         return maxQueryRunningTaskCount;
     }
 
-    @Config("experimental.max-query-running-task-count")
```

Comment:
This very PR is changing the semantics of this field.  Isn't it premature to rename this?

Reply:
@mayankgarg1990  @tdcmeehan   I changed the new config introduced in this PR to `experimental.halt-query-scheduling-when-task-limit-exceeded` and added `@LegacyConfig()` for old ones, which were introduced in v0.225 and would continue to exist.